xml/art_sles_rpi_quick.xml

```diff
@@ -893,8 +892,8 @@ Password:
    </note>
    <para>
     <emphasis role="strong">After you obtained a registration code from a
```

Comment:
"After you obtained" should be "After you obtain".

Reply:
Not my text, so shouldn't block the update please.
Would "After you have obtained..." work, too?

Reply:
yes that works too